assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -42,11 +47,25 @@ class GettingStartedOverview extends React.Component<Props, State> {
     this.setState({color: color.hex});
   };
 
+  async componentWillUnmount() {
```

Comment:
rather than handling this on unmount, it might be better to just auto-save on the "blur" event (i.e. when a user clicks out of an input field)

Reply:
(one reason: i ~think this won't get called if the user closes the window)

Reply:
can we get rid of this now that the blur callbacks are setup?